modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainer.java

```diff
@@ -108,7 +107,7 @@ public MuleContainer(String[] args)
         this.deploymentService = new MuleDeploymentService(artifactResourcesRegistry.getDomainFactory(), artifactResourcesRegistry.getApplicationFactory());
         this.repositoryService = new RepositoryServiceFactory().createRepositoryService();
 
-        this.toolingService = new DefaultToolingService(repositoryService, new TemporaryToolingArtifactBuilderFactory(artifactResourcesRegistry));
+        this.toolingService = new DefaultToolingService(repositoryService, new TemporaryTemporaryArtifactBuilderFactory(artifactResourcesRegistry));
```

Comment:
After further reading, it seems it should be DefaultTemporaryArtifactBuilderFactory